src/arpx/tileInfo/TileInfo.hx

```diff
@@ -6,6 +6,9 @@ import arpx.structs.ArpRange;
 @:arpType("tileInfo")
 class TileInfo implements IArpObject {
 
+	@:arpField public var tileWidth:Float = 32;
+	@:arpField public var tileHeight:Float = 32;
```

Comment:
Note these are pretty arbitrary default values, should be overridden with seed env like `<env name="seed.tileInfo" tileWidth="16" tileHeight="16" />`